R/ml_random_forest.R

```diff
@@ -41,11 +53,41 @@ ml_random_forest <- function(x,
     ml.options = ml.options
   )
 
+  col.sample.rate <- ensure_scalar_double(col.sample.rate, allow.null = TRUE)
```

Comment:
Sorry; I see now this decision was made explicitly. I think it's fine to enforce that this has to be a numeric value.